src/core.js

```diff
@@ -1288,10 +1288,20 @@ define([
                 _this.setUnsavedDuplicateNodeId(false);
             } else if (e.property === 'defaultValue' && e.mug.__className === 'Item') {
                 _this.setUnsavedDuplicateChoiceValue(false);
+            } else if (e.property === 'dataParent') {
+                // TODO Dirty write hack that needs to be replace
+                e.mug.options.isDataOnly = true;
```

Comment:
Did this to make sure the control tree doesn't change. Could also extend this to use another param or refactor this into another method to insert into data only.

Thoughts on what would be better?